chitchat/src/lib.rs

```diff
@@ -34,16 +34,16 @@ use crate::state::ClusterState;
 /// Map key for the heartbeat node value.
 pub(crate) const HEARTBEAT_KEY: &str = "heartbeat";
 
-/// Maximum payload size (in bytes) for UDP.
+/// Maximum UDP datagram payload size (in bytes).
 ///
-/// Note 60KB typically won't fit in a UDP frame,
-/// so long message will be sent over several frame.
+/// Note that 65KB typically won't fit in a single IP packet,
+/// so long messages will be sent over several IP fragments of MTU size.
 ///
-/// We pick a large MTU because at the moment
+/// We pick a large payload size because at the moment because
 /// we send the self digest "in full".
-/// A frame of 1400B would limit us to 20 nodes
+/// An Ethernet frame size of 1400B would limit us to 20 nodes
 /// or so.
-const MTU: usize = 60_000;
+const MAX_UDP_DATAGRAM_PAYLOAD_SIZE: usize = 65_507;
```

Comment:
> A UDP datagram is carried in a single IP packet and is hence limited to a maximum payload of 65,507 bytes for IPv4 and 65,527 bytes for IPv6

[Source](https://erg.abdn.ac.uk/users/gorry/course/inet-pages/udp.html#:~:text=A%20UDP%20datagram%20is%20carried,and%2065%2C527%20bytes%20for%20IPv6.)